packages/react/src/types/transactions.ts

```diff
@@ -0,0 +1,63 @@
+import { BigNumber } from "@ethersproject/bignumber";
+
+export interface TxReceipt {
+    to: string;
+    from: string;
+    contractAddress: string;
+    transactionIndex: number;
+    blockHash: string;
+    transactionHash: string;
+    blockNumber: number;
+    status?: number;
+}
+
+export enum TxType {
+    CUSTOM,
+    KPI_TOKEN_CREATION,
```

Comment:
where do these variables came from?

Reply:
These are determined on the base of the most used transaction types that we will most likely see in Carrot. I defined them by just knowing what the contracts do gneerally speaking.